extensions/sql-database-projects/src/dialogs/createProjectFromDatabaseDialog.ts

```diff
@@ -106,6 +112,14 @@ export class CreateProjectFromDatabaseDialog {
 								component: createProjectSettingsFormSection,
 							}
 						]
+					},
+					{
+						title: constants.workspace,
+						components: [
```

Comment:
this can be simplified to just component: createWorkspaceContainerFormSection since there's only one component

Reply:
I tried with just "component" instead of components and the title doesn't remain bold. Hence I kept it as-is.
![image](https://user-images.githubusercontent.com/57200045/101288244-63025f80-37aa-11eb-81b8-d5a184452536.png)

Reply:
hmm that's weird. Can you open a bug for this?

Reply:
Sure, I will do that.